diff -uNr libressl-3.4.3/crypto/CMakeLists.txt libressl-3.4.3_patched/crypto/CMakeLists.txt
--- libressl-3.4.3/crypto/CMakeLists.txt	2022-03-13 18:17:47.000000000 +0100
+++ libressl-3.4.3_patched/crypto/CMakeLists.txt	2022-04-14 08:19:58.592059913 +0200
@@ -1004,9 +1004,9 @@
 
 export_symbol(crypto ${CMAKE_CURRENT_BINARY_DIR}/crypto_p.sym)
 target_link_libraries(crypto ${PLATFORM_LIBS})
-if (WIN32)
-	set(CRYPTO_POSTFIX -${CRYPTO_MAJOR_VERSION})
-endif()
+#if (WIN32)
+#	set(CRYPTO_POSTFIX -${CRYPTO_MAJOR_VERSION})
+#endif()
 set_target_properties(crypto PROPERTIES
 	OUTPUT_NAME crypto${CRYPTO_POSTFIX}
 	ARCHIVE_OUTPUT_NAME crypto${CRYPTO_POSTFIX})
diff -uNr libressl-3.4.3/ssl/CMakeLists.txt libressl-3.4.3_patched/ssl/CMakeLists.txt
--- libressl-3.4.3/ssl/CMakeLists.txt	2022-03-13 18:17:47.000000000 +0100
+++ libressl-3.4.3_patched/ssl/CMakeLists.txt	2022-04-14 08:19:39.004043509 +0200
@@ -72,9 +72,9 @@
 
 export_symbol(ssl ${CMAKE_CURRENT_SOURCE_DIR}/ssl.sym)
 target_link_libraries(ssl crypto ${PLATFORM_LIBS})
-if (WIN32)
-	set(SSL_POSTFIX -${SSL_MAJOR_VERSION})
-endif()
+#if (WIN32)
+#	set(SSL_POSTFIX -${SSL_MAJOR_VERSION})
+#endif()
 set_target_properties(ssl PROPERTIES
 	OUTPUT_NAME ssl${SSL_POSTFIX}
 	ARCHIVE_OUTPUT_NAME ssl${SSL_POSTFIX})
diff -uNr libressl-3.4.3/tls/CMakeLists.txt libressl-3.4.3_patched/tls/CMakeLists.txt
--- libressl-3.4.3/tls/CMakeLists.txt	2022-03-13 18:17:47.000000000 +0100
+++ libressl-3.4.3_patched/tls/CMakeLists.txt	2022-04-14 08:20:11.592070015 +0200
@@ -57,9 +57,9 @@
 
 export_symbol(tls ${CMAKE_CURRENT_BINARY_DIR}/tls.sym)
 target_link_libraries(tls ${PLATFORM_LIBS})
-if (WIN32)
-	set(TLS_POSTFIX -${TLS_MAJOR_VERSION})
-endif()
+#if (WIN32)
+#	set(TLS_POSTFIX -${TLS_MAJOR_VERSION})
+#endif()
 set_target_properties(tls PROPERTIES
 	OUTPUT_NAME tls${TLS_POSTFIX}
 	ARCHIVE_OUTPUT_NAME tls${TLS_POSTFIX})
